subsystem/kubeapi_test.go

```diff
@@ -1167,6 +1183,73 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		checkAgentCondition(ctx, host.ID.String(), v1beta1.BoundCondition, v1beta1.UnboundReason)
 	})
 
+	It("Agent labels", func() {
+		By("Deploy InfraEnv")
+		infraEnvSpec.ClusterRef = nil
+		deployInfraEnvCRD(ctx, kubeClient, infraNsName.Name, infraEnvSpec)
+
+		infraEnvKubeName := types.NamespacedName{
+			Namespace: Options.Namespace,
+			Name:      infraNsName.Name,
+		}
+
+		By("Verify ISO URL is populated")
+		Eventually(func() string {
+			return getInfraEnvCRD(ctx, kubeClient, infraEnvKubeName).Status.ISODownloadURL
+		}, "15s", "5s").Should(Not(BeEmpty()))
```

Comment:
5 seconds is a long time, you can probably change it to every second

Reply:
Done